client/app/bundles/course/assessment/submission/components/ScribingView/__test__/SavingIndicator.test.js

```diff
@@ -21,6 +21,7 @@ const mockSubmission = {
     attemptedAt: '2017-05-11T15:38:11.000+08:00',
     basePoints: 1000,
     canGrade: true,
+    showPublicTestCasesOutput: true,
```

Comment:
Is this intended? you are passing the flag to the scribing questions. I think only programming questions need the flag.

Reply:
Maybe I got confused somewhere, that's why I had included the flag here as well. Will remove flags from these files